Lib/logging/handlers.py

```diff
@@ -815,7 +815,10 @@ def __init__(self, address=('localhost', SYSLOG_UDP_PORT),
 
         if isinstance(address, str):
             self.unixsocket = True
-            self._connect_unixsocket(address)
+            try:
```

Comment:
There's [`contextlib.suppress`][suppress] which you may enjoy if you like context managers :)

[suppress]: https://docs.python.org/3/library/contextlib.html#contextlib.suppress